### ISSUE TITLE
broccoli-dist-es6-module#mattia

### ISSUE THREAD

**Issue by givanse** [unspace/un-calendar#9]
Couldn't the project have the custom files within itself instead of depending on the customization of another package? I'm talking about:

```
git+https://github.com/ghedamat/broccoli-dist-es6-module#mattia/use-main-name-for-all-builds
```

Say, I want to add the `repository` field to avoid the ugly warnings:

```
npm WARN package.json broccoli-dist-es6-module@0.2.1 No repository field.
```

I don't want to customize the customization of a dependency.

**Comment by ghedamat** [comment]
@givanse we're a few months late here :/ sorry!

somehow I didn't get any of these notifications.

we could probably do that you suggest, we use a customized version of `broccoli-dist-es6-module` because the original version was left unmantained and we needed that particular feature

I guess we could copy the source within our project and remove the dependency altogether. although at this point I think it would be best to spend the time and port this whole project to an ember-cli addon

**Comment by givanse** [comment]
Yep, time has passed and an addon its probably the best route now.

**Comment by givanse** [comment]
If the addon route is taken, tip: make two addons, one for the calendar component and another one for the CSS.
